src/Negotiation/Negotiator.php

```diff
@@ -49,11 +49,17 @@ protected function match(AcceptHeader $accept, AcceptHeader $priority, $index)
         list($acceptSub, $acceptPlus) = $this->splitSubPart($acceptSub);
         list($prioritySub, $priorityPlus) = $this->splitSubPart($prioritySub);
 
+        // If no wildcards in either the subtype or + segment, do nothing.
+        if (! ($acceptBase === '*' || $baseEqual)
+            || ! ($acceptSub === '*' || $prioritySub === '*' || $acceptPlus === '*' || $priorityPlus === '*')) {
```

Comment:
According to PSR-2 when the if is multi-line the `) {` needs to be on it's own line.